src/pages/docs/infrastructure/workers/autoscaling-workers/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-08-22
+modDate: 2024-08-22
+title: Auto Scaling
+navOrder: 60
+---
+
+Workers are only utilized during the execution of a Deployment Process - which means they need to be able to handle high
+workloads intermittently but remain idle otherwise.
+
+Workers installed on virtual or physical machines require minimal resources during quiet times, leaving the machine 
+under-utilized for much of its life.
+
+Using the Kubernetes Worker resolves this issue. You can install it in a Kubernetes cluster, which leverages the cluster's
+ability to scale hardware as workloads change.
+
+Auto-scaling benefits aside, the Kubernetes Worker _is_ a standard Octopus worker:
+* It must be included in 1 or more worker pools
+* Supports deployments to any deployment target
+* Will be kept up to date via machine health checks & updates
+* Can execute operations in custom containers (as defined on the deployment step)
```

Comment:
I'd cut this. I don't imagine users would have any reason to expect that k8s workers don't do this right?